src/main/kotlin/no/nav/dagpenger/regel/api/arena/adapter/RegelApiArenaAdapter.kt

```diff
@@ -177,6 +178,13 @@ fun Application.regelApiAdapter(
             )
             call.respond(status, problem)
         }
+
+        exception<SubsumsjonProblem> { cause ->
+            call.respond(HttpStatusCode.BadGateway, cause.problem).also {
```

Comment:
I am not sure what statuscode to return...

Reply:
i think bad gateway is ok